templates/style/_navbar.scss

```diff
@@ -73,18 +79,18 @@ div.nav-container {
         }
     }
 
-    .pure-menu-right {
-        float: right;
-    }
-
     form.landing-search-form-nav {
         max-width: 1200px;
+        height: 100%;
```

Comment:
I think you can get rid of this `height: 100%`. That makes the search field vertically misaligned, but you can fix that by also removing the `height: 100%` on line 92.

Reply:
Without the `height: 100%`:

![Screenshot from 2022-04-13 16-56-07](https://user-images.githubusercontent.com/3050060/163209459-89b28e6f-ce27-4086-a185-3f1c397ad228.png)